pages/slice/language-guide/class-types.md

```diff
@@ -178,6 +178,25 @@ When the generated code decodes a class in sliced format and slices off derived
 slices: it keeps them in the decoded instance. If you later send this class instance to another application that knows
 the derived class type, this application will decode successfully the full type.
 
+## Compact type IDs
+
+When the generated code encodes a class it needs to transmit the class's type.
+It does this by encoding the class's _type ID_ - a unique value associated with each class.
+Normally, this is the class's fully-qualified identifier.
+However, to reduce overhead, you can assign a numeric type ID to a class instead. For example:
+
+```slice
+module MyModule
+{
+    class ClassWithStringTypeId {}     // has a type ID of "MyModule::ClassWithStringTypeId"
+    class ClassWithNumericTypeId(9) {} // has a type ID of `9`
+}
+```
+
+These _compact_ type IDs must be non-negative and unique across your application.
```

Comment:
I find putting compact in italics is not necessary.

Reply:
Removed the italics.